Hyperspace.podspec

```diff
@@ -8,7 +8,7 @@
 
 Pod::Spec.new do |s|
   s.name             = 'Hyperspace'
-  s.version          = '3.0.0'
+  s.version          = '3.1.0'
```

Comment:
One additional thing I'd say is that we normally will pool a bunch of changes together before creating a new release, so we should revert this to 3.0.0 for now.